component/src/governance/check.rs

```diff
@@ -87,6 +92,41 @@ pub mod stateless {
         Ok(())
     }
 
+    pub fn delegator_vote(
+        DelegatorVote {
+            auth_sig,
+            proof,
+            body:
+                DelegatorVoteBody {
+                    start_position,
+                    value,
+                    nullifier,
+                    rk,
+                    // Unused in stateless checks:
+                    vote: _,            // Only used when executing the vote
+                    proposal: _,        // Checked against the current open proposals statefully
+                    unbonded_amount: _, // Checked against the proposal's snapshot exchange rate statefully
+                },
+        }: &DelegatorVote,
+        context: Arc<Transaction>,
+    ) -> Result<()> {
+        let effect_hash = context.transaction_body().effect_hash();
+        let anchor = context.anchor;
+
+        // 1. Check spend auth signature using provided spend auth key.
+        rk.verify(effect_hash.as_ref(), auth_sig)
+            .context("delegator vote auth signature failed to verify")?;
+
+        // 2. Check that the proof verifies.
+
+        // Verify the proof with this position as the start position:
```

Comment:
What is "this position" here, as distinct from "start position"? Does this comment mean something different from "Verify the proof with the provided anchor and start position"?